### ISSUE TITLE
Failed building: spawn ./configure ENOENT

### ISSUE THREAD

**Issue by kennethlynne** [jaredallard/nexe#167]
Tried running it for the first time and got this output

 ```bash
$ nexe -i ./input.js -o ./output

----> framework => nodejs
----> downloading https://nodejs.org/dist/node-latest.tar.gz
[==========]
----> tar -xf /script/tmp/nexe/nodejs/latest/nodejs-latest.tar.gz -C /script/tmp/nexe/nodejs/latest
----> bundle /script/invite.js
----> Browserify finished
----> embedResources undefined
----> resource -> /script/tmp/nexe/nodejs/latest/node-v5.5.0/lib/nexeres.js
----> monkey patch /script/tmp/nexe/nodejs/latest/node-v5.5.0/node.gyp
----> monkey patch /script/tmp/nexe/nodejs/latest/node-v5.5.0/src/node.js
----> monkey patch /script/tmp/nexe/nodejs/latest/node-v5.5.0/lib/child_process.js
----> make
events.js:142
      throw er; // Unhandled 'error' event
      ^

Error: spawn ./configure ENOENT
    at exports._errnoException (util.js:856:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:179:32)
    at onErrorNT (internal/child_process.js:345:16)
    at nextTickCallbackWith2Args (node.js:455:9)
    at process._tickCallback (node.js:369:17)

```

**Comment by jaredallard** [comment]
Fixed in nexe@1.0.5 I forgot to publish it!

**Comment by jaredallard** [comment]
Also, duplicate of #166. But oh well!
